docs/contracts/v4/guides/15-single-sided-liquidity.mdx

```diff
@@ -0,0 +1,423 @@
+The actual **amount** of `token0` and `token1` that a Pool owes the Position owner is calculated from the parts of the liquidity position that are to the left and right of the current Tick.
+Liquidity left of the current Tick is denominated in `token0` and liquidity right of the current Tick is denominated in `token1`.
+
+If a new Position is created and the Tick Range of the position does not include the current Tick of the Pool, only one of the two Tokens in the Pool can be provided.
+
+We will call this a **Single Side Liquidity Position**.
+
+<img src={require('../images/range-order.png').default} alt="RangeOrder" box-shadow="none"/>
+
+When the current Tick of the Pool moves across the Position, the ratio of `token0` and `token1` will change, and ultimately inverse if the current Tick moves out of the position on the other side.
+
+We will utilise this behaviour to provide liquidity with `token1` and withdraw the position when it has been converted to `token0`.
+
+## Calculating the Tick Range
+
+Our goal for this guide is to create a [Buy Limit Order](https://docs.uniswap.org/concepts/protocol/range-orders#buy-limit-orders) that trades `token1` for `token0` when the Price of `token1` decreases by 5%.
+
+To create our Position, we need to first decide the Tick Range that we want to provide liquidity in.
+
+### Upper Tick
+
+We [create a Pool](https://docs.uniswap.org/contracts/v4/quickstart/create-pool) that represents the v4 Pool we are interacting with and get the `token1Price`.
+We won't need full tick data in this example.
+
+```solidity
+import {TickMath} from "v4-core/src/libraries/TickMath.sol";
+import {Constants} from "./base/Constants.sol";
+import {Config} from "./base/Config.sol";
+
+uint24 public constant LP_FEE = 3000;
+int24 public constant TICK_SPACING = 60;
+uint256 public constant token1Amount = 1e6; //USDC
+
+PoolKey memory pool = PoolKey({
+    currency0: currency0, // imported from Config.sol
+    currency1: currency1, // imported from Config.sol
+    fee: LP_FEE,
+    tickSpacing: TICK_SPACING,
+    hooks: hookContract
+});
+
+// Get current pool sqrtPrice and tick
+(uint160 sqrtPriceX96,,,) = POOLMANAGER.getSlot0(pool.toId()); //POOLMANAGER is imported from Constants.sol
+int24 currentTick = TickMath.getTickAtSqrtPrice(sqrtPriceX96);
+```
+
+Next we decrease the `Price` by 5%. We create a new Price with a numerator 5% lower than our current Price:
+
+```solidity
+uint160 targetSqrtPriceX96 = uint160((uint256(sqrtPriceX96) * 974679) / 1e6); //Since we need to calculate 95% of the current sqrtprice
+```
+
+We have calculated our target Price but we still need to find the **nearest usable tick** to create our Position.
+
+:::info
+As Positions can only start and end at initializable Ticks of the Pool, so we can only create a Range Order to a Price that exactly matches an initializable Tick.
```

Comment:
this line is confusing to me but i know what youre trying to say

something like this is a lot more clear ot me:
"A position's ticks should align with the pool's tick spacing. If a pool's tick spacing is 60, the position's tickLower and tickUpper should be a multiple of 60, i.e. [-300, 180]"